src/rezplugins/shell/cmd.py

```diff
@@ -7,7 +7,7 @@
 from rez.system import system
 from rez.utils.system import popen
 from rez.utils.platform_ import platform_
-from rez.util import shlex_join
+from rezplugins.shell.powershell import list2cmdline
```

Comment:
This is nasty! If there's common functionality here then it should be moved into util code somewhere. There certainly should not be cross referencing of shell implementations like this